### PR TITLE
feat: Header control atom was added

### PR DESCRIPTION
**Pull request by tukoski**
screencapture-localhost-6006-2023-08-23-23_11_23

### CODE REVIEW

File: themes/color.ts [hunk 1]
```diff
@@ -83,8 +82,12 @@ export const Color = {
   colorMenuDarkBg: "#bae0ff",
   colorMenuSubItemBg: "#bae0ff",
 
+  // table
+  tableHeaderBg: "#F5F6F8",
```
Comment (Cromeror) [inline themes/color.ts:86]: We need to delete this token and use the [ `colorBgLayout` ](https://github.com/Cromeror/Fanatics-Design-System/blob/f40af4029d19180955c616c980147e509c46781d/themes/color.ts#L73) token

File: themes/color.ts [hunk 2]
```diff
@@ -83,8 +82,12 @@ export const Color = {
...
+  tableHeaderBorder: "#00000014",
```
Comment (Cromeror) [inline themes/color.ts:87]: We need to delete this token and use the [ `colorSplit`](https://github.com/Cromeror/Fanatics-Design-System/blob/f40af4029d19180955c616c980147e509c46781d/themes/color.ts#L89) token. Afterward, we need to correct the color in the color palette. Please replace its value with #25292F0F

**_Note_**: Remember run the script `antd2scss`

File: themes/color.ts [hunk 3]
```diff
@@ -137,146 +140,146 @@ export const Color = {
...
+  red1: "#fff1f0",
+  red2: "#431418",
+  red3: "#58181c",
+  red4: "#791a1f",
+  red5: "#a61d24",
+  red6: "#d32029",
+  red7: "#cf1322",
+  red8: "#a8071a",
+  red9: "#f89f9a",
+  red10: "#fac8c3",
```
Comment (Cromeror) [inline themes/color.ts:152]: Great job!! 💥

File: components/headerControl/headerControl.module.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+@import "themes/antd.tokens";
+
+.container {
+    display: flex;
+    justify-content: center;
+    background-color: $tableHeaderBg;
+    width: 32px;
+    height: 36px;
```
Comment (Cromeror) [inline components/headerControl/headerControl.module.scss:8]: If the padding, margins, and sizes are correct, we don't need to adjust the height and width.

File: components/headerControl/headerControl.module.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    border-bottom: 1px solid $tableHeaderBorder;
```
Comment (Cromeror) [inline components/headerControl/headerControl.module.scss:9]: Use the token [colorSplit](https://github.com/Cromeror/Fanatics-Design-System/blob/f40af4029d19180955c616c980147e509c46781d/themes/color.ts#L89) here!
```suggestion
    border-bottom: 1px solid $colorSplit;
```

File: themes/color.ts [hunk 6]
```diff
@@ -83,8 +82,12 @@ export const Color = {
   colorMenuDarkBg: "#bae0ff",
   colorMenuSubItemBg: "#bae0ff",
 
+  // table
+  tableHeaderBg: "#F5F6F8",
+  tableHeaderBorder: "#00000014",
+
   //Border y Drivers
-  colorBorder: "#25292F29",//16%
+  colorBorder: "#25292F29", //16%
```
Comment (Cromeror) [inline themes/color.ts:90]: Feel free to remove the comment like this 🍃

Comment (tukoski) [reply themes/color.ts]: Updated

File: components/headerControl/headerControl.module.scss (hunk 5 above)
Comment (tukoski) [reply components/headerControl/headerControl.module.scss]: Updated

File: themes/color.ts (hunk 3 above)
Comment (tukoski) [reply themes/color.ts:148]: Updated

File: themes/color.ts (hunk 2 above)
Comment (tukoski) [reply themes/color.ts]: Updated

**Review by Cromeror — APPROVED** [review]
🚀

**Pull request merged**